test/rlcIntegration/generated/bodyStringRest/package.json

```diff
@@ -107,5 +107,11 @@
   },
   "browser": {
     "./dist-esm/test/public/utils/env.js": "./dist-esm/test/public/utils/env.browser.js"
+  },
+  "//sampleConfiguration": {
+    "productName": "",
```

Comment:
Why is this being generated as empty string?

Reply:
Due to the description in package detail is empty and will improve this in next pr